src/functions/heroFunctions/deleteHero.tsx

```diff
@@ -0,0 +1,6 @@
+import { PLAYERS_KEY } from "../../typesAndConstants/constants";
+
+export default function deleteHero(players: { [key: string]: HeroToJSON }, player: string) {
```

Comment:
cant this be a method  in the base hero? also have the same for create and update of hero. so that the hero class manages the saving deleting updating. or even have characters manager class.